### PR TITLE
[ADD] stock_no_negative v9

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
@alexis-via @jbeficent Deport the stock_no_negative from v10 to v9
cc @darshan-serpent

### CODE REVIEW

**Comment by serpentcs-dev1** [comment]
@jbeficent Could you please review?

**Review by MiquelRForgeFlow — APPROVED** [review]
tested on runbot :+1:

File: stock_no_negative/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Stock Disallow Negative
```
Comment (MiquelRForgeFlow) [inline stock_no_negative/README.rst:6]: Stock Disallow Negative → Stock No Negative

File: stock_no_negative/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015-2016 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+{
+    'name': 'Stock Disallow Negative',
```
Comment (MiquelRForgeFlow) [inline stock_no_negative/__openerp__.py:8]: Stock Disallow Negative → Stock No Negative

**Review by MiquelRForgeFlow — COMMENTED** [review]
Should have the true name of the module.

File: stock_no_negative/README.rst (hunk 1 above)
Comment (serpentcs-dev1) [reply stock_no_negative/README.rst:6]: @mreficent Hmm.. I think it is right. As this module is deported from [v10 PR](https://github.com/OCA/stock-logistics-workflow/pull/275) over there it was like this.

Maybe @jbeficent knows better. As there's already a module in v8 with the same name, right?

And if still there's any change needed please let me know, I'll do it.

**Review by Cedric-Pigeon — APPROVED** [review]

**Review by lmignon — APPROVED** [review]
:+1:  Code review

**Pull request merged**